plugins/aggregators/quantile/README.md

```diff
@@ -0,0 +1,135 @@
+numbers of samples and is therefor **recommended**.
+
+For implementation details see the underlying [a golang library][tdigest_lib].
+
+##### exact R7
+This type "exactly" computes the quantiles according to [Hyndman & Fan (1996) R7][hyndman_fan]
+algorithm. This variant is also used in Excel and NumPy.
+
+This approximator needs to store all seen data in the aggregation `period` and
+might consume a lot of memory when used with a large number of series or a
+large number of samples. Furthermore, it is very *slow* compared to the `t-digest`
+approximator and therefore **should only be used with small number of samples**.
+
+##### exact R8
+This type "exactly" computes the quantiles according to [Hyndman & Fan (1996) R8][hyndman_fan]
+algorithm. This variant is recommended by Hyndman & Fan in their paper due to
+its independence of the underlying sample distribution.
+
+This approximator needs to store all seen data in the aggregation `period` and
+might consume a lot of memory when used with a large number of series or a
+large number of samples. Furthermore, it is very *slow* compared to the `t-digest`
+approximator and therefore **should only be used with small number of samples**.
+
+
+#### Benchmark (linux/amd64)
+The benchmark was performed by adding 100 metrics with six numeric
+(and two non-numeric) fields to the aggregator and the derive the aggregation
+result.
+
+| approximator  | # quantiles   | avg. runtime  |
+| :------------ | -------------:| -------------:|
+| t-digest      |            3  |  376372 ns/op |
+| exact R7      |            3  | 9782946 ns/op |
+| exact R8      |            3  | 9158205 ns/op |
+| t-digest      |          100  |  899204 ns/op |
+| exact R7      |          100  | 7868816 ns/op |
+| exact R8      |          100  | 8099612 ns/op |
+
+### Measurements
+Measurement names are passed trough this aggregator.
+
+### Fields
+
+For all numeric fields (int32/64, uint32/64 and float32/64) new *quantile*
+fields are aggregated in the form `<fieldname>_<quantile*100>`. Other field
+types (e.g. boolean, string) are ignored and dropped from the output.
+
+For example passing in the following metric as *input*:
+- somemetric
+  - average_response_ms (float64)
+  - minimum_response_ms (float64)
+  - maximum_response_ms (float64)
+  - status (string)
+  - ok (boolean)
+
+and the default setting for `quantiles ` you get the following *output*
+- somemetric
+  - average_response_ms_025 (float64)
+  - average_response_ms_050 (float64)
+  - average_response_ms_075 (float64)
+  - minimum_response_ms_025 (float64)
+  - minimum_response_ms_050 (float64)
+  - minimum_response_ms_075 (float64)
+  - maximum_response_ms_025 (float64)
+  - maximum_response_ms_050 (float64)
+  - maximum_response_ms_075 (float64)
+
+As you can see `status` and `ok` are dropped due to their type. Furthermore,
+the number of resulting fields scales with the number of `quantiles` specified.
```

Comment:
```suggestion
The `status` and `ok` fields are dropped because they are not numeric.  Note that the
number of resulting fields scales with the number of `quantiles` specified.
```